cg/models/cg_config.py

```diff
@@ -730,3 +734,18 @@ def delivery_rsync_service(self) -> DeliveryRsyncService:
             )
             self.delivery_rsync_service_ = service
         return service
+
+    @property
+    def delivery_service_factory(self) -> DeliveryServiceFactory:
+        factory = self.__dict__.get("delivery_service_factory_")
```

Comment:
```suggestion
        factory = self.delivery_service_factory_
```
Is there a difference?